docs/about/team/internships.md

```diff
@@ -4,6 +4,19 @@ sidebar_position: 4
 
 # Internships at the AI Lab
 
+## [Samuel Pelletier-Evraire](www.linkedin.com/in/samuel-pelletier-evraire) - Application mobile hivers 2024
```

Comment:
Only the month is necessary for the `internships.md` file:
Hiver 2024
The LinkedIn link is not working, you need to add `https:\\` at the beginning